source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,133 @@
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* addons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+A partial snapshot consists of any number of the above default directories and installed add-ons.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not. Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+5. Click "Create" to begin the snapshot.
+
+### Restoring a Snapshot on a new install
+
+You can restore a snapshot during on a new installation during the onboarding process. Follow the link at the bottom of the account creation page to upload your snapshot from a previous installation.
+
+For restoring a snapshot at any other time, vist the Supervisor Snapshot panel in your UI and use the following steps:
+
+1. Select "Upload Snapshot" from the icon in the upper right of the page.
+1. Click on the folder icon to navigate to your snapshot .tar file
+
+When the upload is completed, you will be presented with the snapshot restore dialog for restoring it, and can then choose to restore in full or in part by manually selecting individual items.
+
+If a snapshot is significantly large, it can be faster and more efficient to make use of the Samba add-on in order to transfer files to the `/backup` directory. 
+
+
+### How long do snapshots take?
+
+The length of time it takes to create or restore snapshots will depend on how much you have to compress or decompress. 
+
+A large database file or a collection of camera media captured by MotionEye will take longer to decompress (the default database is the sqllite home-assistant_v2.db in the config directory). See the [`recorder`](https://www.home-assistant.io/components/recorder/) integration page for options to keep your database data down to a size that won't cause issues. Note the keep days, purge interval, and include/exclude options.
+
+When the restore is complete, you will lose connection to the UI and it will return just as though you had restarted Home Assistant.
+
+### Creating snapshots using the custom CLI
+
+1. `ha sn list` - lists snapshots and their slugnames
+2. `ha sn restore slugname` - restores a specific snapshot
+3. `ha sn new --name nameofsnapshot` - create a snapshot
+
+Use `ha help` to see more info.
+
+
+### Copying your snapshots to another location
+
+The snapshots won't be very convenient if they are on the device which you cannot access. Copy them off in batches manually from /backup on occasion. Or even better, create an automation to handle that, or make use of an add-on like Google Drive backup or Dropbox Sync.
```

Comment:
```suggestion
You usually need a snapshot in case your system has crashed. If you just store them on the crashed device, you won't be able to access it. We recommend that you manually copy them from `/backup` to another machine on occasion. Or even better, create an automation to handle that, or make use of an add-on like Google Drive backup or Dropbox Sync.
```